### ISSUE TITLE
SelectTool drag/drop a View changes ID on View

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1710]
Probably better to assign to @jonferraiolo but assigning to @jhpedemonte for fear @jonferraiolo will be too swamped in short-term.
- New HTML File - composition type of one of the mobile devices
- Drag/drop a View (id set automatically to View_0)
- Drag/drop a second View (id set automatically to View_1
- With View_1 selected, put mouse of selection chrome and drag around a bit and let go

Result=> View_1 is now named View_2.

If you repeat, it becomes View_3. If you put a custom ID onto that View (e.g., 'aaa') and then drag around that view, its ID changes from 'aaa' to 'View_4'

**Comment by jhpedemonte** [comment]
Might be the same as (or related to) #1062.

**Comment by jhpedemonte** [comment]
-> M6

**Comment by jhpedemonte** [comment]
This seems to have been fixed.  For one, I cannot move the Views -- I believe Jon fixed that already.

I tried to do the same by adding a Button, setting the ID and moving it around, but the ID never changed.

Closing.

**Comment by JonFerraiolo** [comment]
As I remember, I actually fixed both the ID changing problem and prevented user from dragging Views that are direct children of BODY (just like we don't allow dragging of ContentPane's that are inside of TabContainer, BorderContainer or Accordion). As I remember, I fixed the ID changing problem first, convinced myself the problem was addressed, and then I disabled View dragging.

Short version: OK to close this bug.
